handlers.go

```diff
@@ -75,6 +75,8 @@ var dcsHandlers = map[int]handlerFn{
 var escHandler = map[int]handlerFn{
 	'7': printf("Save cursor"),
 	'8': printf("Restore cursor"),
+	'>': printf("Normal Keypad (DECPNM)"),
+	'=': printf("Application Keypad (DECPAM)"),
```

Comment:
someone proposed a way of adding mnemonics in a standard way, I think, for the sake of making everything look the same for now, I'd leave them as comments until we do that.

wdyt?

```suggestion
	'>': printf("Normal Keypad"),      // DECPNM
	'=': printf("Application Keypad"), // (DECPAM
```

Reply:
for reference, mnemonics discussion: https://github.com/charmbracelet/sequin/discussions/59

Reply:
Done!